src/ekore/anomalous_dimensions/unpolarized/time_like/as2.py

```diff
@@ -0,0 +1,741 @@
+"""The unpolarized time-like NLO Altarelli-Parisi splitting kernels."""
+
+import numba as nb
+import numpy as np
+
+# from eko.constants import zeta2
+from numpy import power as npp
```

Comment:
Python has a built-in power operator, i.e. `**`. Numba supports it, as much as `numpy.power`, so you will get the same code.

Please, favor built-ins over third-party extension functions (including NumPy).